### DISCUSSION TITLE
my copilot was banned

### DISCUSSION THREAD

**Discussion by dsj320** [community/community#116481] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Github official say that the system detected repeated use of credential sharing services,i can't use copilot any more,why?I think i don't do anything against the rule.

**Comment by Akash1134** [comment]
Hi @dsj320 , I reviewed your account activity and recommend that you contact our [Support team](https://support.github.com/contact). Please open a ticket with them :)

**Reply by dsj320** [reply, marked as answer]
Thank you for your response. I have opened a ticket with the official team, but they haven't replied to me yet. I hope they can restore my Copilot accessibility.
